cosmo_tester/test_suites/image_based_tests/test_tenant_creation_rabbitmq_down.py

```diff
@@ -1,29 +1,24 @@
 from cloudify_rest_client.exceptions import CloudifyClientError
-from cosmo_tester.framework.fixtures import (  # noqa
-    image_based_manager_without_plugins,
-)
 
-manager = image_based_manager_without_plugins
 
-
-def test_tenant_creation_no_rabbitmq(manager):
-    with manager.ssh() as fabric:
-        fabric.sudo('systemctl stop cloudify-rabbitmq')
+def test_tenant_creation_no_rabbitmq(image_based_manager):
+    image_based_manager.run_command(
+        'systemctl stop cloudify-rabbitmq', use_sudo=True)
 
     try:
-        manager.client.tenants.create('badtenant')
+        image_based_manager.client.tenants.create('badtenant')
```

Comment:
this is better written as `with pytest.raises(CloudifyClientError)`

Reply:
So it is- will clean it up while we're here.